kmip/core/factories/secrets.py

```diff
@@ -68,7 +69,7 @@ def create(self, secret_type, value=None):
             SymmetricKey(...)
         """
         if secret_type is ObjectType.CERTIFICATE:
-            return self._create_certificate(value)
+            return self._create_certificate()
```

Comment:
OK, I can create the certificate object directly. In Barbican the API takes in optional parameters for algorithm and crypto lengths. I assume those are for the public key in the certificate. I'm not sure. I'll ignore those for now because Barbican Core will save them and just have the KMIP secret store store the certificate.